llpc/test/shaderdb/core/OpTypeSampledImage_TestWaterfallScalarize.frag

```diff
@@ -16,18 +13,56 @@ void main()
     _3 = texture(_11[nonuniformEXT(_12)], _6);
 }
 
-// BEGIN_SHADERTEST
-//
-// RUN: amdllpc -scalarize-waterfall-descriptor-loads -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+// RUN: amdllpc -scalarize-waterfall-descriptor-loads -v %gfxip %s | FileCheck -check-prefix=GFX %s
```

Comment:
Why the different test prefix by the way?

Reply:
Done :) Thank you! I did not know that we have a standard name for it.